add ability to set multiple testcases for one test

#### I'm submitting a ... 
  - [ ] bug report
  - [x] feature request
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

#### What is the current behavior?

only one link to test case present in the allure report

#### What is the motivation / use case for changing the behavior?

one test can cover several test cases, it would be nice if we have the same behavior like we have in `pytest-allure-adaptor==1.7.9`

### Example of exception we have now

```
tests/test_landing_page.py:7: in <module>
    @pytest.allure.feature('Landing page')
tests/test_landing_page.py:11: in TestLanding
    @pytest.allure.testcase('39', '43', '748')
E   TypeError: testcase() takes from 1 to 2 positional arguments but 3 were given
```

#### Please tell us about your environment:

- Allure version:        2.5.0
- Test framework:        pytest@3.3.2
- Allure adaptor:        allure-pytest@2.2.4b1

--- a/allure-pytest/src/helper.py
+++ b/allure-pytest/src/helper.py
@@ -12,7 +12,6 @@ class AllureTestHelper(object):
 
     def __init__(self, config):
         self.config = config
-        self.x = 0
 
     @allure_commons.hookimpl
     def decorate_as_title(self, test_title):

--- a/allure-pytest/src/helper.py
+++ b/allure-pytest/src/helper.py
@@ -12,6 +12,7 @@ class AllureTestHelper(object):
 
     def __init__(self, config):
         self.config = config
+        self.x = 0
 
     @allure_commons.hookimpl
     def decorate_as_title(self, test_title):
@@ -36,7 +37,11 @@ class AllureTestHelper(object):
 
     @allure_commons.hookimpl
     def decorate_as_link(self, url, link_type, name):
-        allure_link_marker = '{prefix}.{link_type}'.format(prefix=ALLURE_LINK_PREFIX, link_type=link_type)
+        allure_link_marker = '{prefix}.{link_type}:{postfix}'.format(
+            prefix=ALLURE_LINK_PREFIX,
+            link_type=link_type,
+            postfix=url
+        )
         pattern = dict(self.config.option.allure_link_pattern).get(link_type, u'{}')
         url = pattern.format(url)
         allure_link = getattr(pytest.mark, allure_link_marker)

--- a/allure-pytest/test/links/function_links_test.py
+++ b/allure-pytest/test/links/function_links_test.py
@@ -16,6 +16,7 @@ LINK_TYPE = 'homepage'
 ISSUE = 'https://github.com/qameta/allure-integrations/issues/8'
 ISSUE_NAME = 'Github issue'
 TEST_CASE = 'https://github.com/qameta/allure-integrations/issues/8#issuecomment-268313637'
+ANOTHER_TEST_CASE = 'https://github.com/allure-framework/allure-python/issues/191'
 TEST_CASE_NAME = 'Comment with spec'
 
 
@@ -95,3 +96,16 @@ def test_with_links_cases_and_issues():
     ...             ))
     """
     pass
+
+
+@pytest.allure.testcase(TEST_CASE, name="First")
+@pytest.allure.testcase(ANOTHER_TEST_CASE, name="Second")
+def test_multiply_some_type_links():
+    """
+    >>> allure_report = getfixture('allure_report')
+    >>> assert_that(allure_report,
+    ...             has_test_case('test_multiply_some_type_links',
+    ...                 has_test_case_link(TEST_CASE, name='First'),
+    ...                 has_test_case_link(ANOTHER_TEST_CASE, name='Second')
+    ...             ))
+    """

